src/PsrTarget.php

```diff
@@ -81,7 +88,61 @@ public function export()
                 }
             }
 
-            $this->getLogger()->log($this->_psrLevels[$message[1]], $text, $context);
+            $this->getLogger()->log($this->_levels[$level], $text, $context);
+        }
+    }
+
+    /**
+     * Sets the message levels that this target is interested in.
+     *
+     * The parameter can be an array.
+     * Valid level names include: 'error',
+     * 'warning', 'info', 'trace' and 'profile'; valid level values include:
+     * [[Logger::LEVEL_ERROR]], [[Logger::LEVEL_WARNING]], [[Logger::LEVEL_INFO]],
+     * [[Logger::LEVEL_TRACE]], [[Logger::LEVEL_PROFILE]] and Psr Log levels:
+     * [[LogLevel::EMERGENCY]], [[LogLevel::ALERT]], [[LogLevel::CRITICAL]],
+     * [[LogLevel::ERROR]], [[LogLevel::WARNING]], [[LogLevel::NOTICE]],
+     * [[LogLevel::INFO]] and [[LogLevel::DEBUG]].
+     *
+     * For example,
+     *
+     * ```php
+     * ['error', 'warning', LogLevel::CRITICAL, LogLevel::EMERGENCY]
+     * ```
+     *
+     * @param array $levels message levels that this target is interested in.
+     * @throws InvalidConfigException if $levels value is not correct.
+     */
+    public function setLevels($levels)
+    {
+        static $levelMap = [
```

Comment:
That would disallow using PSR levels in setLevels(). Is that intended?

Reply:
Why? _levels contains the PSR levels. I need to write tests for this.